[2025-10-02T05:22:27.723+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-02T05:22:27.728+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sentiment140_end2end_single.download_raw_from_s3 manual__2025-10-02T05:22:24.782187+00:00 [queued]>
[2025-10-02T05:22:27.731+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sentiment140_end2end_single.download_raw_from_s3 manual__2025-10-02T05:22:24.782187+00:00 [queued]>
[2025-10-02T05:22:27.731+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-02T05:22:27.735+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_raw_from_s3> on 2025-10-02 05:22:24.782187+00:00
[2025-10-02T05:22:27.738+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2334) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-02T05:22:27.738+0000] {standard_task_runner.py:63} INFO - Started process 2336 to run task
[2025-10-02T05:22:27.738+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sentiment140_end2end_single', 'download_raw_from_s3', 'manual__2025-10-02T05:22:24.782187+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/edaproject.py', '--cfg-path', '/tmp/tmpbub25u8a']
[2025-10-02T05:22:27.739+0000] {standard_task_runner.py:91} INFO - Job 18: Subtask download_raw_from_s3
[2025-10-02T05:22:27.754+0000] {task_command.py:426} INFO - Running <TaskInstance: sentiment140_end2end_single.download_raw_from_s3 manual__2025-10-02T05:22:24.782187+00:00 [running]> on host 7be8b7489da9
[2025-10-02T05:22:27.779+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sentiment140_end2end_single' AIRFLOW_CTX_TASK_ID='download_raw_from_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-10-02T05:22:24.782187+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-02T05:22:24.782187+00:00'
[2025-10-02T05:22:27.780+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-02T05:22:27.785+0000] {logging_mixin.py:188} INFO - [download_raw_from_s3] Using CSV_S3_URI=s3://298a1/sentiment140/***/main-data.csv
[2025-10-02T05:22:27.789+0000] {base.py:84} INFO - Using connection ID 'aws_default' for task execution.
[2025-10-02T05:22:27.789+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2025-10-02T05:22:28.750+0000] {s3.py:1387} INFO - Downloading source S3 file from Bucket 298a1 with path sentiment140/***/main-data.csv
[2025-10-02T05:22:36.093+0000] {logging_mixin.py:188} INFO - [download_raw_from_s3] Downloaded to /tmp/raw_input.csv
[2025-10-02T05:22:36.146+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-02T05:22:36.148+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-02T05:22:36.190+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=sentiment140_end2end_single, task_id=download_raw_from_s3, run_id=manual__2025-10-02T05:22:24.782187+00:00, execution_date=20251002T052224, start_date=20251002T052227, end_date=20251002T052236
[2025-10-02T05:22:36.220+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-02T05:22:36.244+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-02T05:22:36.245+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
